src/main/java/com/test/controller/EmployeeController.java

```diff
@@ -1,69 +0,0 @@
-package com.test.controller;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
-
-import com.test.service.EmployeeService;
-import com.test.model.*;
-
-@RestController
-@RequestMapping("/employee")
-public class EmployeeController {
-	
-	@Autowired
-	EmployeeService employeeService;
-	
-	@GetMapping("/all")
-	@RequestMapping(value = "/all",method = RequestMethod.GET)
```

Comment:
@Rinshakp 

**GetMapping**, implies that RequestMapping with the URL and request method of GET.
This is an improper way of using the mappings as both should not exist together.